### DISCUSSION TITLE
Can some of this code be used outside of Krata?

### DISCUSSION THREAD

**Discussion by DemiMarie** [edera-dev/krata#147] [General]
I’m interested in using this code for Qubes OS, to avoid needing QEMU for domains using PCI passthrough.  Could parts of Krata be useful for this?

**Comment by azenla** [comment]
We are in the process of separating out our different components into reusable libraries, so yes we expect it to available to others. On PCI passthrough particularly, we expect to be getting a lot of this into the main branch in the coming weeks!

**Reply by DemiMarie** [reply]
That is amazing!

**Comment by azenla** [comment]
Hi @DemiMarie, in talking with various folks over the last few weeks, we've decided to start working towards making krata more palatable for use outside of the original intent. krata is now our designation for all the open source components in our product that can be used and adopted by Xen users such as Qubes OS.

We will be working on cleaning things up more this week, but please feel free to let us know what you think.

**Reply by DemiMarie** [reply]
That is awesome!  I’m excited for future developments :).
